react-frontend/src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './TextInput.css'
+
+const TextInput = ({ type, placeholder, value, onChange, label }) => {
+  return (
+    <div className="text-input-container">
+      {label && <label className="text-input-label">{label}</label>}
```

Comment:
The same as for other labels

Reply:
Refactored [0c2832f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/0c2832f95bc0ca4eb2cb355a368e619b1378eb2b)